aim/api/status.py

```diff
@@ -58,7 +58,7 @@ class AciStatus(resource.ResourceBase, OperationalResource):
     def __init__(self, **kwargs):
         super(AciStatus, self).__init__({'resource_type': None,
                                          'resource_id': None,
-                                         'sync_status': self.SYNC_PENDING,
+                                         'sync_status': None,
```

Comment:
Not sure I understand the need for this. Is there a real reason for sync_status to be ever unset? It seems useful to AIM users if they can rely on getting a valid sync_status value always.